src/ToDo.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable react/jsx-no-duplicate-props */
+import React from 'react';
+import style from './Todo.module.scss';
+
+const ToDo = ({
+  toDoList, addToDo, removeToDo, resetTodo,
+}) => {
+  const addInputValue = () => {
+    const input = document.getElementsByClassName(`${style.input}`)[0];
+    if (input.value !== '') {
+      addToDo(input.value);
+    }
+    input.value = '';
+  };
+  const removeToDoItem = (className) => {
+    const shouldRemove = document.getElementsByClassName(`${className}`)[0].innerHTML;
+    removeToDo(shouldRemove);
+  };
+  const toDoArr = [];
+  for (let i = 0; i < toDoList.length; i++) {
+    toDoArr.push(toDoList[i].name);
+  }
+  return (
+    <div>
+      <input className={style.input} type="text" placeholder="Write to do..." />
+      <button className={style.submit} type="button" onClick={() => addInputValue()}>Submit</button>
```

Comment:
you don`t need to create one more arrow function to call inner function, just pass your function.
onClick={addInputValues}